inventory/group_vars/k8s-cluster.yml

```diff
@@ -40,7 +40,7 @@ kube_log_level: 2
 
 # Users to create for basic auth in Kubernetes API via HTTP
 # Optionally add groups for user
-kube_api_pwd: "changeme"
+kube_api_pwd: "{{ lookup('password', 'credentials/kube_user length=15') }}"
```

Comment:
may be better using different passwords for users?

Reply:
Yeah, that's not hard

Reply:
I will just get rid of root user. Nobody uses it.